src/cf-forms.less

```diff
@@ -30,6 +31,9 @@
 // .error
 @input-error:                   #ea130b;
 
+// warning
+@input-warning:                 #ffb108;
```

Comment:
Be sure to add this variable to `cf-vars-overrides.less`, if you haven't already, defaulting to the CFPB palette color of choice.